src/server/server.js

```diff
@@ -566,7 +571,8 @@ function tickPlayer(currentPlayer) {
            .reduce( function(a, b, c) { return b ? a.concat(c) : a; }, []);
 
         if(virusCollision > 0 && currentCell.mass > virus[virusCollision].mass) {
```

Comment:
While you're at it, can you fix this check? It should be `virusCollision.length > 0`